speech/speech.py

```diff
@@ -65,7 +65,7 @@ def process_speech_text(text):
 def create_audio(path, text):
     # logging.info(f"Generating Audio File : {text}")
     text = process_speech_text(text)
-    if not os.path.exists(path):
+    if not os.path.exists(path) or not os.path.getsize(path) > 0:
```

Comment:
Have noticed sometimes that if an mp3 file fails to work and writes a zero size mp3 file it will fail subsequent video generations, so this will overwrite an mp3 file if it's 0 bytes in size.